packages/compartment-mapper/test/main.test.js

```diff
@@ -56,25 +50,9 @@ const assertFixture = (t, namespace) => {
     globalLexicals.globalLexical,
     "exports global lexical"
   );
-  t.deepEqual(
-    typecommon,
-    [42, 42, 42, 42],
-    "type=common package carries exports"
-  );
-  t.deepEqual(
-    typemodule,
-    [42, 42, 42, 42],
-    "type=module package carries exports"
-  );
-  t.deepEqual(
-    typeparsers,
-    [42, 42, 42, 42],
-    "parsers-specifying package carries exports"
-  );
-  t.equal(typehybrid, 42, "type=module and module= package carries exports");
 };
 
-const fixtureAssertionCount = 11;
+const fixtureAssertionCount = 6;
```

Comment:
Ever more eager for ava, to make this `t.plan` stuff go away. These manually maintained numbers are awful.

Just editorial venting. No change needed to this PR.